datacube/testutils/io.py

```diff
@@ -132,32 +137,168 @@ def native_geobox(ds: Dataset, measurements=None, basis: str | None = None):
     return geobox
 
 
-def native_load(ds, measurements=None, basis=None, **kw):
-    """Load single dataset in native resolution.
+def compute_native_load_geobox(
+    ds: Dataset,
+    band: str,
+    dst_geobox: GeoBox | None = None,
+    buffer: float | None = None,
+) -> GeoBox:
+    """Compute area of interest for an input dataset given final output geobox.
+
+    Take native projection and resolution from ``ds, band`` pair and compute
+    region in that projection that fully encloses footprint of the
+    ``dst_geobox`` with some padding. Construct GeoBox that encloses that
```

Comment:
Is "padding" in this line the same as the parameter called "buffer"? If they are the same, it would be good to always use the same term when referring to them.